### DISCUSSION TITLE
Branch tab is not showing

### DISCUSSION THREAD

**Discussion by pspallepitiya** [community/community#129428] [GitHub Education]
### Hi everyone,

I just created a GitHub account using GitHub web page. I wanted to create a new branch, but I cannot see branch Tab. I really appreciate your help. Thank you.

**Comment by manuel14mds** [comment]
Hi @pspallepitiya !
To create a new branch on GitHub, you'll typically follow these steps. If you're not seeing the "Branch" tab or option, it's possible that you're not in the correct repository view or you might be missing some permissions. Here's a detailed guide to help you create a new branch:
Step-by-Step Guide to Create a New Branch on GitHub

    Navigate to Your Repository:
        Go to github.com/
        Log in to your account if you haven't already.
        Navigate to the repository where you want to create the new branch.

    Access the Branch Dropdown:
        Once you're in the repository, you should see the "main" or "master" branch button near the upper left corner, above the list of files.
        Click on this branch button. It usually shows the current branch name (e.g., "main" or "master").

    Create a New Branch:
        In the dropdown that appears, you'll see a text box with the option to "Find or create a branch..."
        Type the name of the new branch you want to create in this text box.
        As you type, an option to create a branch from the typed name will appear (e.g., "Create branch your-branch-name").
        Click on this option to create the new branch.

    Switch to the New Branch:
        After creating the new branch, GitHub will automatically switch to it.
        You can now make changes and commits in this new branch.

Troubleshooting: If You Don't See the Branch Option

    Check Permissions:
        Ensure you have the necessary permissions to create branches in the repository. If you're a collaborator, you should typically have this permission. If not, contact the repository owner or admin.

    Repository View:
        Make sure you're in the main view of the repository. If you're in a specific file or commit view, you might not see the branch options.

    Refresh the Page:
        Sometimes, simply refreshing the page can solve the issue if the page didn't load correctly.
